certsubmitter/certsubmitter.go

```diff
@@ -85,6 +70,34 @@ func Run(ctx context.Context, lc *client.LogClient, ca *certgen.CA, sv *ct.Signa
 	glog.Infof("%s: %s: stopped", l.URL, logStr)
 }
 
+func issueAndSubmit(ctx context.Context, lc *client.LogClient, ca *certgen.CA, st storage.APICallWriter, l *ctlog.Log) ([]*x509.Certificate, *ct.SignedCertificateTimestamp, error) {
+	glog.Infof("%s: %s: issuing certificate chain...", l.URL, logStr)
+	var chain []*x509.Certificate
+	var err error
+	chain, err = ca.IssueCertificateChain()
+	if err != nil {
+		glog.Errorf("%s: %s: ca.IssueCertificateChain(): %s", l.URL, logStr, err)
+		return nil, nil, err
+	}
+
+	glog.Infof("%s: %s: adding chain...", l.URL, logStr)
+	var sct *ct.SignedCertificateTimestamp
+	var httpData *client.HTTPData
+	var addErr error
+	sct, httpData, addErr = lc.AddChain(chain)
```

Comment:
Why did this change from `sct, httpData, addErr := lc.AddChain(chain)`, any reason for that? I normally try to avoid separately declaring variables (just another thing to maintain), but sometimes have to, is this the case here?

Reply:
Ack